recipe/meta.yaml

```
@@ -1,16 +1,13 @@
 {% set name = "dbt-semantic-interfaces" %}
-{% set version = "0.7.4" %}
+{% set version = "0.9.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_') }}-{{ version }}.tar.gz
```

```suggestion
  - url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_') }}-{{ version }}.tar.gz
```
Also, should it be a list if only one source available (I understand that it doesn't matter in grand scheme of things)?